specs/util/decoders.spec.js

```diff
@@ -46,13 +45,12 @@ const SUBSTRATE_ID = new Uint8Array([0x53]);
 const CRYPTO_SR25519 = new Uint8Array([0x01]);
 const CMD_SIGN_MORTAL = new Uint8Array([0]);
 const CMD_SIGN_MSG = new Uint8Array([3]);
+const registry = new TypeRegistry();
+new MetaData(registry, kusamaData);
```

Comment:
This new part is not nice, and trigger an eslint warning of `do not use new with side effect`, but before figure it out with Jaco, let us use this to fix the current problem.